stable/artifactory-ha/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
+## [4.3.3] - Oct 15, 2020
```

Comment:
@AshutoshNirkhe Small correction needed , Add a line space  before the changelog entry

Reply:
@chukka done